codec/proto/internal/proto/test.proto

```
@@ -0,0 +1,10 @@
+syntax = "proto3";
+
+package protobuf;
```

Buf (1.54.0)

Files with package "protobuf" must be within a directory "protobuf" relative to root but were in directory "codec/proto/internal/proto".

(PACKAGE_DIRECTORY_MATCH)